### PR TITLE
Add MicroPets (BEP20)

### PR DESCRIPTION
**Pull request by micropets2021**
Adding info for BEP20 token 'MicroPets'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by micropets2021** [comment]
We had to change our github to an organization for our entire team. This Pull request is just for a logo change for us

**Comment by micropets2021** [comment]
We notice that there is a merge error due to "conflicting files" the old github we used had to be changed to a organization for our entire development team to have access to it. This Pull Request is just for a logo change. Hope this isnt too much of an issue. Hope to hear back, thank you.
________________________________

**Comment by Cryptocool1** [comment]
submission looks good, waiting for the fee payment

File: blockchains/smartchain/assets/0xA77346760341460B42C230ca6D21d4c8E743Fa9c/info.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            "url": "https://github.com/micropets2021"
+        },
+        {
+            "name": "coingecko",
+            "url": "https://coingecko.com/en/coins/micropets"
+        }
+    ],
+    "tags": [
+        "nft",
+        "defi,",
```
Comment (unanoc) [inline blockchains/smartchain/assets/0xA77346760341460B42C230ca6D21d4c8E743Fa9c/info.json:35]: remove a comma from `defi,`

**Pull request merged**